README.md

```diff
@@ -19,30 +19,6 @@ defineCustomElements(window);
 <manifold-oauth oauth-url="[your-oauth-url]"></manifold-oauth>
 ```
 
-4. Listen for the event `tokenReceived`:
-
-```js
-document.addEventListener("tokenReceived", e => {
-  console.log(e.detail); // received token
-  // do stuff...
-});
-```
```

Comment:
These instructions weren’t accurate; currently it only emits events perceptible to other Stencil libraries.

Reply:
Well, that's not true, is it? You're right that the event doesn't bubble to the document, but you can listen for the event on the element. It should probably just be changed to show a query for the `manifold-oauth` component and do `component.addEventListener` instead.

Reply:
Well the event name is wrong, too. It’s `receiveManifoldToken`, but I had to dig into the Netlify preview app which wasn’t named correctly.

Reason I’m being a stickler is it’s not for me; it’s for our partners. If the docs are wrong, then as far as they’re concerned it’s broken software. This is broken.

Reply:
Sure but does that mean the documentation should be removed, or fixed?